src/core/diff/strategies/new-unified/__tests__/edit-strategies.test.ts

```diff
@@ -275,7 +277,7 @@ describe("applyGitFallback", () => {
 		expect(result.result.join("\n")).toEqual("line1\nnew line2\nline3")
 		expect(result.confidence).toBe(1)
 		expect(result.strategy).toBe("git-fallback")
-	})
+	}, 10_000)
```

Comment:
So this implies that whatever we’re doing with git in the experimental diff algo is taking between 5 and 10 seconds?

Reply:
I encountered this timeout while debugging, and increasing it seemed to suddenly make the problems go away. Undoing this change seemed to make the problem reliably come back. One theory I had was that it impacted the order or sequencing of the tests such that the failure stopped happening. I added the --randomize flag to jest to test that theory but we get a bunch of other failures since some of our tests do seem to rely on being run in a particular order (which we should fix).